accounting/accounting/doctype/stock_ledger_entry/stock_ledger_entry.json

```diff
@@ -0,0 +1,79 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2023-09-25 15:02:57.919313",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "item",
+  "warehouse",
+  "entry_time",
+  "quantity",
+  "rate"
```

Comment:
This should refer to the document that created it, else there's no clear linkages. 


Since more than one document type can create ledger, use "dynamic link".